scripts/templates/README.md

```diff
@@ -0,0 +1,20 @@
+# {{ theme_title }}
+
+{{ theme_description }}
+
+![Screenshot of the {{ theme }} theme in a bash script](./images/{{ theme }}.png)
+
+## Colors
+
+Background color: ![#{{ background_hex }}](https://via.placeholder.com/20/{{ background_hex }}/{{ background_hex }}.png) `#{{ background_hex }}`
+
+Highlight color: ![#{{ highlight_hex }}](https://via.placeholder.com/20/{{ highlight_hex }}/{{ highlight_hex }}.png) `#{{ highlight_hex }}`
+
+**WCAG compliance**
+
+| Color | Hex | Ratio | Normal text | Large text |
+| ----- | --- | ----- | ----------- | ---------- |
```

Comment:
I think we should be able to precompute the length of each columns and use `| format`.

I thinks it is also often too easy to reach for jinja when we don't need to.

It might be easier to generate this table from python and use `{{table}}` here instead of bending backward. I can give it a try.

Reply:
Is the idea here to not use a Jinja template at all? I'm fine with that, I just wasn't sure what alternative would be acceptable

Reply:
@trallard and I talked about me removing the Jinja dependency and putting this in a Python template string instead

Reply:
@Carreau are you currently hacking on this or should I go ahead?

Reply:
I moved the table generation in the the script.

Reply:
Sorry, I didn't wanted to overstep so did not try to hack it, I'll try to be more explicit next time. Also side question, is that ok for me to push on your branches to help things move forward ?

Reply:
Yes, that's completely fine. Just please don't force push unless absolutely necessary.